src/2_storage.js

```diff
@@ -61,7 +61,17 @@ var retrieveValue = function(value) {
 };
 
 var webStorage = function(perm) {
-	var storageMethod = perm ? localStorage : sessionStorage;
+	var storageMethod;
+	try {
+		storageMethod = perm ? localStorage : sessionStorage;
```

Comment:
No, an attempt to access either localStorage or sessionStorage when storage is disabled is throwing an Error.  Even just trying to assign either to storageMethod in this case is throwing the error.